cypress/support/commands.js

```diff
@@ -13,7 +13,7 @@ import { whenReady } from './utils';
 //
 // -- This is a parent command --
 
-Cypress.Commands.add('login', () => {
+const login = () => {
```

Comment:
Extracted this out from the login helper as it's used by both `login` and `loginNoCallback`.